subprojects/docs/src/docs/userguide/multi_project_builds.adoc

```diff
@@ -48,403 +48,660 @@ Configuration on demand mode attempts to configure only projects that are releva
+└── settings.gradle
+----
+
+[source.multi-language-sample,groovy]
+.settings.gradle
+----
+include::{samplesPath}/userguide/multiproject/dependencies/webDist/settings.gradle[]
+----
+
+[source.multi-language-sample,groovy]
+.build.gradle
+----
+include::{samplesPath}/userguide/multiproject/dependencies/webDist/build.gradle[]
+----
 
 We have an interesting set of dependencies. Obviously the `date` and `hello` projects have a _configuration_ dependency on `webDist`, as all the build logic for the webapp projects is injected by `webDist`. The _execution_ dependency is in the other direction, as `webDist` depends on the build artifacts of `date` and `hello`. There is even a third dependency. `webDist` has a _configuration_ dependency on `date` and `hello` because it needs to know the `archivePath`. But it asks for this information at _execution time_. Therefore we have no circular dependency.
 
 Such dependency patterns are daily bread in the problem space of multi-project builds. If a build system does not support these patterns, you either can't solve your problem or you need to do ugly hacks which are hard to maintain and massively impair your productivity as a build master.
 
 [[sec:project_jar_dependencies]]
-=== Project lib dependencies
-
-What if one project needs the jar produced by another project in its compile path, and not just the jar but also the transitive dependencies of this jar? Obviously this is a very common use case for Java multi-project builds. As already mentioned in <<sub:project_dependencies>>, Gradle offers project lib dependencies for this.
-
-++++
-<sample id="javadependencies_1" dir="userguide/multiproject/dependencies/java" includeLocation="true" title="Project lib dependencies">
-            <layout>
-                settings.gradle
-                build.gradle
-                api/
-                api/src/main/java/
-                api/src/main/java/org/gradle/sample/api/Person.java
-                api/src/main/java/org/gradle/sample/apiImpl/PersonImpl.java
-                services/personService/
-                services/personService/src/main/java/
-                services/personService/src/main/java/org/gradle/sample/services/PersonService.java
-                services/personService/src/test/java/
-                services/personService/src/test/java/org/gradle/sample/services/PersonServiceTest.java
-                shared/
-                shared/src/main/java/
-                shared/src/main/java/org/gradle/sample/shared/Helper.java
-            </layout>
-        </sample>
-++++
+== Project lib dependencies
+
+What if one project needs the jar produced by another project in its compile path, and not just the jar but also the transitive dependencies of this jar? Obviously this is a very common use case for Java multi-project builds. As mentioned in <<dependency_types.adoc#sub:project_dependencies,Project dependencies>>, Gradle offers project lib dependencies for this.
+
+.Project lib dependencies
```

Comment:
❌ Missing note admonition about where to find the sample in the -all distribution, same as above